tests/system/test_resource_crud.py

```diff
@@ -50,3 +48,27 @@ def test_path_methods(identity):
     actual = identity.user_path("bdfl")
 
     assert expected == actual
+
+
+def test_nonslash_resource(messaging):
+    expected = "users/bdfl/profile/blurbs/legacy/apocalyptic~city"
```

Comment:
Out of curiosity, was does the `~` mean in a path?

Reply:
It's used in certain ads resources for reasons I don't understand. In this instance right here it's just intended to be not a slash.